src/platforms/shared.rs

```diff
@@ -82,3 +81,21 @@ pub fn shared_build_opts_ccache(use_ccache: bool) -> Vec<String> {
         vec![]
     }
 }
+
+/// Ignore duplicate libraries warnings for MacOS with XCode>=15.
```

Comment:
Let's use surrounding `///` lines for functions.

Reply:
Done.

Reply:
Please don’t merge all commits into one while the review is still in process.  Clicking on the “[View it on GitHub](https://github.com/matter-labs/era-compiler-llvm-builder/pull/24/files/d1c2a61ffad69c2852af119fcc7c3daa61ec8257..afc751850972f2796a5281455fd9ecbd4df6e195)” link in the notification email now gives:

> We went looking everywhere, but couldn’t find those commits.
> Sometimes commits can disappear after a force-push.

It’s a lot easier to review changes purely in comments than the whole thing over again.  :-) 

(Eventually I found them via the [hash link](https://github.com/matter-labs/era-compiler-llvm-builder/commit/afc751850972f2796a5281455fd9ecbd4df6e195) in the email:  “This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.”)